### DISCUSSION TITLE
Application unable to write files

### DISCUSSION THREAD

**Discussion by gili-gili** [microsoft/win32-app-isolation#27] [Q&A] [answered]
The application I'm going to package needs to write something under `C:\Program Files\WindowsApps\*AppName*`
I found [this](https://learn.microsoft.com/en-us/windows/uwp/packaging/app-capability-declarations) but it doesn't seem to have the capability I want.
Any ideas?

**Comment by ddsilva-msft** [comment, marked as answer]
Hi @gili-gili, we don't allow applications to write under the application install directory as that can result in tampering with application binaries and configuration. We do allow access to a folder under C:\ProgramData containing the publisher ID in the name e.g. C:\ProgramData\AppData_m1z3e72mn4yei where m1z3e72mn4yei is the publisher ID. This access is allowed implicitly without any capabilities.
